lib/archethic/p2p/message.ex

```diff
@@ -1227,12 +1227,9 @@ defmodule ArchEthic.P2P.Message do
   end
 
   def process(%GetFirstPublicKey{address: address}) do
```

Comment:
We should also rename `address` to `public_key`, don't you think ?

Reply:
Sure sam, that makes more sense.

Reply:
but we are supplying address to get a public key for a given address right

Reply:
so address also is a better argument name, also %GetFirstPublicKey{} accepts a address: , so variable name of address make more sense.

Reply:
Hi, please keep it as public key, and in comments put address explaining it .

Reply:
The name is misleading, the purpose of the function
So in GetFirstPublicKey, the public key should have more meaning.
Moreover, the implementation underneath is not using the address at all

Reply:
Okay thanks, I think it's more clear now

Reply:
I have made the changes.